docs/source/guide/ml.md

```diff
@@ -16,74 +16,79 @@ With Label Studio, you can set up your favorite machine learning models to do th
+
+If you need to load static pre-annotated data into Label Studio, running an ML backend might be more than you need. Instead, you can [import pre-annotated data](predictions.html).
+
 ## Quickstart
 
 Get started with a machine learning (ML) backend with Label Studio. You need to start both the machine learning backend and Label Studio to start labeling. You can review examples in the [`label-studio-ml/examples` section of the repository](https://github.com/heartexlabs/label-studio-ml/examples).
 
 Follow these steps to set up an example text classifier ML backend with Label Studio:
 
-0. Clone the Label Studio Machine Learning Backend git repository.
-   ```bash
-   git clone https://github.com/heartexlabs/label-studio-ml-backend 
+1. Clone the Label Studio Machine Learning Backend git repository.
+  ```bash
+   git clone https://github.com/heartexlabs/label-studio-ml-backend  
    ```
    
-1. Set up the environment.
+2. Set up the environment.
    ```bash
    cd label-studio-ml-backend
    pip install -e .
    cd label_studio_ml/examples
    pip install -r requirements.txt
    ```
    
-2. Create a new ML backend using the example simple text classifier included in the repository. 
+3. Initialize an ML backend based on an example script:
    ```bash
-   label-studio-ml init my_ml_backend --script label_studio-ml/examples/simple_text_classifier.py
+   label-studio-ml init my_ml_backend --script label_studio_ml/examples/simple_text_classifier.py
    ```
+   This ML backend is an example provided by Label Studio. See [how to create your own ML backend](mlbackend.html).
    
 3. Start the ML backend server.
    ```bash
-   label-studio-ml start my_ml_backend
+   label-studio start text_classification_project --init --template text_classification --ml-backends http://localhost:9090
    ```
    
-4. Start Label Studio and connect your project to the running ML backend using the Label Studio UI. 
-
-
-## Tutorials
-
-For other example machine learning model setups with Label Studio, see the following tutorials:
-- [Create a simple ML backend](/tutorials/dummy_model.html)
-- [Text classification with Scikit-Learn](/tutorials/sklearn-text-classifier.html)
-- [Transfer learning for images with PyTorch](/tutorials/pytorch-image-transfer-learning.html)
-- [Image Object Detector](/tutorials/object-detector.html)
-- [Chatbot response generation with HuggingFace's GPT2 model](/tutorials/gpt.html)
-- [Automatic Speech Recognition with Nvidia's NeMo](/tutorials/nemo_asr.html)
-
-## Getting predictions from a machine learning model
-
-After connecting a model as a machine learning backend, you see model predictions in the labeling interface and on the Tasks page that you use to manage your data.
+4. Start Label Studio and add the link to your machine learning model backend in the **Machine Learning** section of the project settings page.
```

Comment:
We need to have one step before this: setup labeling config in project settings (in this case it should be text classification with Choices + Text)